index.html

```diff
@@ -3,6 +3,12 @@
 ---
 
 <div class="container">
-    {% include resume.html %}
+
+<p> MICCA is the <a href="https://www.contraceptiveaccess.org/">Michigan Collaborative for Contraceptive Access</a>.</p> 
+<p> This site contains tools and resources for performance reporting in the collaborative.</p> 
+<p>     </p> 
+<p>     </p> 
+<p>     </p> 
```

Comment:
I think the html element for breaks is <br>.  Could use a `<div class="myspacer"></div>` and define the spacing the sass.  Not sure what the best practice is for adding vertical space.